src/components/layout/PrimaryLayout/PrimaryLayout.module.css

```diff
@@ -110,7 +110,7 @@
 
 .alpha-flag-page {
   composes: alpha-flag;
-  top: 8px;
+  top: 80px;
```

Comment:
It looks like this flag changes placement a bit between mobile and tablet&desktop. Are the designs the same between these three main layouts? - We should probably consider more responsive stytling here.

![image](https://user-images.githubusercontent.com/30459942/88363334-4a3a6580-cd45-11ea-8bb7-1cbda09f525b.png)

![image](https://user-images.githubusercontent.com/30459942/88363340-545c6400-cd45-11ea-9bf9-0cf829cb5513.png)

Reply:
Yeah, technically this isn't implemented for desktop yet and there's a separate ticket for that, but I don't like how broken this looks so will fix this one

Reply:
The latest version of these designs don't have the top Alpha Flag at all (which makes sense to me, seemed redundant) so I just removed it completely